octodns/provider/base.py

```diff
@@ -57,19 +52,43 @@ def _process_desired_zone(self, desired):
         for record in desired.records:
             if record._type == 'PTR' and len(record.values) > 1:
                 # replace with a single-value copy
-                self.log.warn('does not support multi-value PTR records; '
-                              'will use only %s for %s', record.value,
-                              record.fqdn)
+                self.supports_warn_or_except('does not support multi-value '
+                                             'PTR records; will use only {} '
+                                             'for {}'.format(record.value,
+                                                             record.fqdn))
```

Comment:
NBD, but this message can be confusing to show in the exception. Same for Route53's NA-CA-* message.